Fast Pair: Separate show/hide UI indications from advertising mode (#11887)
nrfconnect/sdk-nrf: samples/bluetooth/peripheral_fast_pair/README.rst

       * After a Bluetooth Central successfully pairs.

       After the device reaches the maximum number of paired devices (:kconfig:option:`CONFIG_BT_MAX_PAIRED`), the device stops looking for new peers.

>        Therefore, the device no longer advertises in the pairing mode (:c:member:`bt_le_adv_prov_adv_state.pairing_mode`), and the Fast Pair advertising mode is automatically set to the Fast Pair not discoverable advertising with the hide UI indication.

Reviewer: ```suggestion
       Therefore, the device no longer advertises in the pairing mode (:c:member:`bt_le_adv_prov_adv_state.pairing_mode`), and only the Fast Pair not discoverable advertising with the hide UI indication can be selected.
```

>        Therefore, the device no longer advertises in the pairing mode (:c:member:`bt_le_adv_prov_adv_state.pairing_mode`), and the Fast Pair advertising mode is automatically set to the Fast Pair not discoverable advertising with the hide UI indication.

Reviewer: `... UI indication on every advertising start.`
